leakpro/attacks/mia_attacks/abstract_mia.py

```diff
@@ -57,7 +58,6 @@ def __init__(
             AbstractMIA._initialized = True
 
         # These objects are instance specific
-        self.logger = handler.logger
```

Comment:
Is there no logger for AbstractMIA now? or is it inherited so its not needed to import?

Reply:
Everywhere where we use the logger, it is imported from the common logger in utils/logger.py. In this particular file, the logger was imported to be used in classes that inherits from abstractMIA. Now, they instead import it directly.